extensions-core/namespace-lookup/src/main/java/io/druid/query/extraction/namespace/URIExtractionNamespace.java

```diff
@@ -373,7 +375,7 @@ public TSVFlatDataParser(
       );
       final DelimitedParser delegate = new DelimitedParser(
           Optional.fromNullable(Strings.isNullOrEmpty(delimiter) ? null : delimiter),
-          Optional.<String>absent()
+          Optional.fromNullable(Strings.isNullOrEmpty(listDelimiter) ? null : listDelimiter)
```

Comment:
(optional) this could be cleaner by doing `Optional.fromNullable(Strings.emptyToNull(listDelimiter))`